client/me/help/help-contact-form/index.jsx

```diff
@@ -77,7 +77,7 @@ export const HelpContactForm = React.createClass( {
 			howYouFeel: 'unspecified',
 			message: '',
 			subject: '',
-			siteSlug: this.getSiteSlug()
+			siteSlug: this.getSiteId()
```

Comment:
Oh I missed that too. It's already merged so I'd suggest just creating a new branch off of latest master.